static/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -33,6 +35,27 @@ import {Widget} from './types';
 import WidgetCardChart from './widgetCardChart';
 import WidgetQueries from './widgetQueries';
 
+type SizeSelectorProps = {
+  size: string;
+  onSizeChange: (size: string) => void;
+};
+
+export const SizeSelector = ({size, onSizeChange}: SizeSelectorProps) => {
```

Comment:
can remove this export if it's only being used in this file!

Reply:
Ah, thanks for catching that! Forgot to remove it when I was putting the component in the same file